src/amo/components/Header/index.js

```diff
@@ -160,9 +182,18 @@ export const mapStateToProps = (state) => {
     api: state.api,
     isHomePage: state.viewContext.context === VIEW_CONTEXT_HOME,
     siteUser: getCurrentUser(state.users),
+    hasAnyReviewerRelatedPermission:
```

Comment:
This seems like it's be a good helper (or possibly even an attribute to set on the user inside the reducer).

At the very least could we abstract this into a helper?

Reply:
Yeah, I wasn't sure initially, but you're right, I'll do that.